pr-data.csv

```diff
@@ -1379,7 +1379,6 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.exec.MockedStructuresTest.testDropTable,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.StopCalciteModuleTest.testStopQueryOnNodeStop,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.basic.TestSortedIndexMvStorageTest.testAbort,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.basic.TestSortedIndexMvStorageTest.testAbortRemove,ID,,,
```

Comment:
I don't see the test removed in https://github.com/apache/ignite-3/commit/188e63f2f62e51f2233554e917ad786f2b3418d4#diff-3f7b6777901a87b1db5df474e42da7013b4a2a6c994b9e04e7f4af18e337203d or even in the abstract superclass https://github.com/apache/ignite-3/commit/188e63f2f62e51f2233554e917ad786f2b3418d4#diff-1f9781c65344f4bfa9a0d554bfbaff5e567a2961b84ff4583835de7be448592b

Do you have the exact line that shows that the test was removed? For now, you can keep this line as is, and handle the other two, and then resolve this later.

Reply:
Sorry. Made a typo here. The test was removed at commit 7e7ab4689db96ea83b11ec2233707645699e6683, and got renamed at 188e63f commit. See screenshot below.
![image](https://user-images.githubusercontent.com/89400432/187089630-13fef4c3-26ed-4f1e-98e2-fe41789f4105.png)

Reply:
Please don't send screenshots but send text or links to text. That shows the test was deleted. It's not a confirmation that it wasn't simply renamed/moved rather than completely deleted.

Reply:
I have searched thru the repo and commit, and made sure that the test is deleted but not moved to somewhere else. I have pushed another commit to the branch. Please see the latest commit, thank you.